InternetStoreCHIReloaded/BLL/Models/Cart.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BLL.Models
+{
+    public class Cart
+    {
+        public int UserId { get; set; }
+
+        // Cart items:
+        public ICollection<ProductWithQuantity> CartItems { get; set; }
```

Comment:
It's really not obvious that class and db named ProductWithQuantity when property named CartItems.
Please, rename ProductWithQuantity to CartItems

Reply:
done